openchs-server-api/src/main/java/org/openchs/web/OrganisationConfigSearchController.java

```diff
@@ -0,0 +1,54 @@
+package org.openchs.web;
+
+import org.openchs.framework.security.UserContextHolder;
+import org.openchs.service.AddressLevelService;
+import org.openchs.service.OrganisationConfigService;
+import org.openchs.web.request.AddressLevelContract;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@RestController
+public class OrganisationConfigSearchController {
+
+    private final OrganisationConfigService organisationConfigService;
+    private final Logger logger;
+    private final AddressLevelService addressLevelService;
+
+    @Autowired
+    public OrganisationConfigSearchController(OrganisationConfigService organisationConfigService, AddressLevelService addressLevelService) {
+        this.organisationConfigService = organisationConfigService;
+        logger = LoggerFactory.getLogger(this.getClass());
+        this.addressLevelService=addressLevelService;
+    }
+
+
+
+    @GetMapping(value = "/web/organisationSearchConfig")
```

Comment:
I think the name of the route is wrong since it gets complete organisationConfig not just search config. We can name it as /web/organisationConfig since that name is not taken by anything else yet.